laterpay/application/Core/Bootstrap.php

```diff
@@ -199,6 +202,11 @@ public function run() {
             add_action( 'laterpay_invoice_indicator',   array( $invoice_controller, 'the_invoice_indicator' ) );
             add_action( 'wp_enqueue_scripts',           array( $invoice_controller, 'add_frontend_scripts' ) );
 
+            // add account links action
+            $account_controller = new LaterPay_Controller_Account( $this->config );
+            add_action( 'laterpay_account_links',       array( $account_controller, 'add_account_links' ), 10, 4 );
```

Comment:
Why is the function called ADD_account_links here? render_account_links sounds more appropriate to me.